test/SqlExtension.Tests/SqlOutputBindingTests.cs

```diff
@@ -40,5 +40,41 @@ public async Task TestAddAsync()
             };
             await collector.AddAsync(data);
         }
+
+        [Fact]
+        public void TestGetTableAndSchema()
+        {
+            string fullName = "dbo.Products";
+            SqlBindingUtilities.GetTableAndSchema(fullName, out string quotedSchema, out string quotedTableName);
+            Assert.Equal("'dbo'", quotedSchema);
+            Assert.Equal("'Products'", quotedTableName);
+        }
+
+        [Fact]
+        public void TestGetTableAndSchemaWithNoSchema()
+        {
+            string fullName = "Products";
+            SqlBindingUtilities.GetTableAndSchema(fullName, out string quotedSchema, out string quotedTableName);
+            Assert.Equal("SCHEMA_NAME()", quotedSchema);
+            Assert.Equal("'Products'", quotedTableName);
+        }
+
+        [Fact]
+        public void TestGetTableAndSchemaWithBrackets()
+        {
+            string fullName = "[dbo].[Products]";
```

Comment:
Just for sanity purposes, please add a test case where only the schema is square bracketed and another where only the table name is bracketed